config/environments/development.rb

```diff
@@ -35,7 +35,12 @@
 
   # Don't care if the mailer can't send.
   config.action_mailer.raise_delivery_errors = false
-
+  host = "localhost:3000"
+  # Use this on the cloud IDE.
+  config.action_mailer.default_url_options = { host: host, protocol: 'http'}
```

Comment:
```suggestion
  config.action_mailer.default_url_options = { host: 'localhost:3000', protocol: 'http'}
```
hạn chế đặt biến trong config vì mấy file config nó load cùng lúc run app